Search/TextSearch.py

```diff
@@ -17,7 +17,7 @@ class firstTextResult(abstractResult):
 
 	def display(self):
 		if len(self.res) != 0:
-			print(self.query + " has the words in common with " + self.res[0])
+			print(self.query + " has the words in common with " + str(self.res[0]))
```

Comment:
I made a mistake, the res should have strings inside, with the corresponding fileName, because the search object has got the reverse index with the correspondingFile function. I will correct it really soon.